corehq/util/argparse_types.py

```diff
@@ -22,25 +22,38 @@ def date_type(value):
         )
 
 
-def validate_integer(gt=None, lt=None):
-    """Return argparser action to validate integer inputs:
-
-    parser.add_argument('--count', action=validate_integer(gt=0, lt=11), help="Integer between 1 and 10")
+# NOTE: The `validate_integer` function has been renamed to `validate_range`,
```

Comment:
I don't think this note is necessary.

Reply:
It's a "grep note" for people who remember using `validate_integer` and go looking for it

Reply:
Sorry I should have been more clear. I was the only one who ever uses that which is why I don't think it's necessary. Fine to leave it though.